src/errors/config-validation-error.ts

```diff
@@ -0,0 +1,12 @@
+import UserError from './user-error';
+import {ValidationError} from 'class-validator';
+
+export default class ConfigValidationError extends UserError {
+  constructor(errors: ValidationError[]) {
+    const messages = errors
+      .map(e => `validation error on ${e.property} with value '${e.value}'`)
+      .join('\n');
+    super(`config validation: ${messages}`);
```

Comment:
Should this be displayed as an array?
```
✖  error     config validation: child "layout" fails because ["layout" is not allowed to be empty]
```

Reply:
Changed to this:

![image](https://user-images.githubusercontent.com/13304739/94552246-dcd01700-0256-11eb-83e3-71b0fb7f49e3.png)